metrics/coverage.py

```
@@ -0,0 +1,104 @@
+
+    columns = {}
+    start = 0
+    for match in re.finditer(r"\s\s+|\n", header):
+        fr, to = match.span()
+        field = header[start:fr]
+
+        value = (0, 0)
+        if field == "Filename":
+            value = (start, to)
+        else:
+            value = (start, fr)
```

Ruff (0.11.9)

Use ternary operator `value = (start, to) if field == "Filename" else (start, fr)` instead of `if`-`else`-block

Replace `if`-`else`-block with `value = (start, to) if field == "Filename" else (start, fr)`

(SIM108)